app/controllers/api/v1/projects_controller.rb

```diff
@@ -75,18 +75,22 @@ def content_from_params(ps)
   end
 
   def build_resource_for_create(create_params)
-    admin_allowed(:approved, :redirect)
+    admin_allowed(:beta_approved, :launch_approved, :redirect)
     create_params[:project_contents] = [ProjectContent.new(content_from_params(create_params))]
     add_user_as_linked_owner(create_params)
     super(create_params)
   end
 
   def build_update_hash(update_params, id)
-    admin_allowed(:approved, :redirect)
+    admin_allowed(:beta_approved, :launch_approved, :redirect)
     content_update = content_from_params(update_params)
     unless content_update.blank?
       Project.find(id).primary_content.update!(content_update)
     end
+    if update_params[:live] == false
```

Comment:
I'm not sure I understand what's unclear about the the project state in that case. The state is that the project isn't live, and the user has requested review for both launching and beta. All of that information can be presented in whatever interface we have for reviewing projects and the reviewer can determine what needs to happen next. 

If we need to keep track of the history of what's happened to project (ie that is has had a beta period) that seems like an entirely separate issue.